indigo/updatedata.py

```diff
@@ -325,6 +336,68 @@ def filter_values(
     return data
 
 
+def get_sandbox_only_data(
+    data,
+    keys_with_own_status_subfield=[],
+    lists_with_items_with_own_status_subfield=[],
+):
+    sandbox_data = {}
+
+    # Some single values (or groups of single values) wight be removed, based on a status field.
```

Comment:
_might_ be removed?

Reply:
I'll fix tomorrow, thanks

Reply:
Also these comments were copied and pasted and aren't actually correct. Which also means there is a typo further up the file too.